docker/falco/Dockerfile

```diff
@@ -1,67 +1,36 @@
-FROM debian:bookworm
+FROM cgr.dev/chainguard/wolfi-base
```

Comment:
Is there any compelling reason not to use a multi-stage build here? :thinking: 

The original [docker/no-driver/Dockerfile.distroless](https://github.com/falcosecurity/falco/pull/3273/files#diff-150a50aeb7255c76b8e25342f071be4d122cc40f1441edc16d4530c319c39078) did so

Reply:
It would make sense if we decided to drop `jq`, `curl` and `ca-certificates` packages from the final image; but we decided to keep them since they are useful for Falco output_program.